### ISSUE TITLE
Add support for Jsonnet to Pygments

### ISSUE THREAD

**Issue by davidzchen** [google/jsonnet#83]
It would be nice to add Jsonnet highlighting to Pygments so that we can have syntax highlighting for Jsonnet code in Markdown fenced code blocks in static site generators such as Jekyll, which is also used for GitHub pages:

<pre>
```jsonnet
{
  local Foo = {
    bar: "Some jsonnet code here"
  },
  baz: Foo {
    bar: "nicely syntax highlighted"
  }
}
```
</pre>

**Comment by mikedanese** [comment]
Also adding a "grammar" to https://github.com/github/linguist would give syntax highlighting in github file view.

**Comment by davidzchen** [comment]
#23 tracks adding Jsonnet support to Linguist, and we sent a pull request github/linguist#2653, but at the time, there were not enough Jsonnet files in GitHub repositories.

Perhaps we should revisit that PR in the near future.

**Comment by sh0rez** [comment]
https://github.com/github/linguist/pull/4455 has been merged nearly a year ago, I guess we can close this!

**Comment by sbarzowski** [comment]
@sh0rez Github support is there, but AFAICT we don't have support in Pygments (see: https://pygments.org/languages/).

**Comment by sh0rez** [comment]
Oh right, this was about Pygments as well. Ignore me 🤪

**Comment by bgagnon** [comment]
Jsonnet is supported by Rouge (demo: http://rouge.jneen.net/v3.19.0/jsonnet/) and there are IDE-specific implementations for VSCode, Jetbrains and Atom. But no Pygments :(

Personally, I'd like to have this for [Mkdocs](https://squidfunk.github.io/mkdocs-material/extensions/codehilite/).

**Comment by AndrewEckart** [comment]
This was completed in https://github.com/pygments/pygments/pull/2239

The change was released in Pygments version 2.14: https://pygments.org/docs/changelog/#version-2-14-0
